virus/Obeliscus_faecalis.md

```diff
@@ -0,0 +1,250 @@
+
+### Virus Genome {Q3}
+
+{Insert your Virus Genome Map Figure and figure legend here}
+
+```         
+Figure 1.1: Index Obelisk Genome Visualization (Obeliscus faecalis 
+Obelisk_000840_000001_00001)
+
+The linearized genome of Obeliscus faecalis Obelisk_000840_000001_00001 (~1023 
+nucleotides) is shown, with predicted open reading frames (ORFs) annotated 
+based on Prodigal analysis. Complete ORFs are represented in blue, while 
+partial ORFs are shown in red. ORF-1 encodes a 210-amino-acid protein and is 
+complete, while ORF-2 encodes a truncated 65-amino-acid protein. The black 
+line depicts the full genome length, and nucleotide coordinates are displayed 
+along the x-axis. Amino acid scales corresponding to the ORFs are provided 
+below the genome. While Obelisks are circular in structure, the genome is 
+presented in linear form for clarity and to highlight the precise positions of 
+ORFs (similar conventions apply to Figures 2.1 and 2.2). This visualization 
+underscores the conserved genomic organization and protein-coding potential of 
+O. faecalis.
+```
+
+![](img/Obeliscus_faecalis/Obelisk_000840_000001_000001_visualization.png)
```

Comment:
Best practice is to provide "Figure Titles" so in the case the image is not loading then text (Figure Title) is at least displayed.